OpenRA.Mods.Common/Warheads/FlashTargetsInRadiusWarhead.cs

```diff
@@ -0,0 +1,60 @@
+#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.GameRules;
+using OpenRA.Mods.Common.Effects;
+using OpenRA.Primitives;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Warheads
+{
+	[Desc("Used to trigger a FlashPostProcessEffect trait on the world actor.")]
```

Comment:
```suggestion
	[Desc("Trigger a flash effect on the targeted actor, or actors within a circle.")]
```
Was this a placeholder until you were clear on if the area effect should be split? If so, it can't hurt to have a comment as a reminder.

Reply:
Yeah I forgot about that part, thanks